tests/test_datastore.py

```diff
@@ -84,3 +106,9 @@ def test_WorldcatQuery_tbl_repr():
         )
         == "<WorldcatQuery(nid='1', resourceId='2', libraryId='1', match='False', responseCode='404')>"
     )
+
+
+def test_datastore_connection(test_connection):
+    with does_not_raise():
+        engine = create_engine(test_connection)
+        engine.connect()
```

Comment:
It would be great to add a test for `session_scope` her as well, I think it's the only thing you've added that's not tested.

Reply:
Done.